src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.html

```diff
@@ -1,79 +1,108 @@
+      <li class="list-group-item">
+        <cd-card-row [data]="rgwBucketCount"
+                     link="/rgw/bucket"
+                     title="Bucket"
+                     summaryType="simplified"
+                     *ngIf="rgwBucketCount != null"></cd-card-row>
+      </li>
+      <hr>
+      <li class="list-group-item">
+        <cd-card-row [data]="UserCount"
+                     link="/rgw/user"
+                     title="User"
+                     summaryType="simplified"
+                     *ngIf="UserCount != null"></cd-card-row>
+      </li>
+      <hr>
+      <li class="list-group-item">
+        <cd-card-row [data]="objectCount"
+                     title="Object"
+                     summaryType="simplified"
+                     *ngIf="objectCount != null"></cd-card-row>
+      </li>
+    </cd-card>
+    <cd-card cardTitle="Performance Statistics"
+             i18n-title
+             class="col-sm-6 d-flex"
+             ria-label="Performance Statistics card">
+      <div class="ms-4 me-4 mt-0">
+        <cd-dashboard-time-selector (selectedTime)="getPrometheusData($event)">
+        </cd-dashboard-time-selector>
+        <cd-dashboard-area-chart chartTitle="Requests/sec"
+                                 dataUnits=""
+                                 label="Requests/sec"
+                                 [data]="queriesResults.RGW_REQUEST_PER_SECOND">
+        </cd-dashboard-area-chart>
+        <cd-dashboard-area-chart chartTitle="Latency"
+                                 dataUnits="ms"
+                                 label="GET"
+                                 label2="PUT"
+                                 [data]="queriesResults.AVG_GET_LATENCY"
+                                 [data2]="queriesResults.AVG_PUT_LATENCY">
+        </cd-dashboard-area-chart>
+        <cd-dashboard-area-chart chartTitle="GET Bandwidth"
+                                 dataUnits="B"
+                                 label="GET Bandwidth"
+                                 [data]="queriesResults.GET_BANDWIDTH">
+        </cd-dashboard-area-chart>
+        <cd-dashboard-area-chart chartTitle="PUT Bandwidth"
+                                 dataUnits="B"
+                                 label="Bandwidth"
+                                 [data]="queriesResults.PUT_BANDWIDTH">
+        </cd-dashboard-area-chart>
+      </div>
+    </cd-card>
     <div class="col-lg-3">
       <cd-card cardTitle="Used Capacity"
                i18n-title
-               class="col-sm-2 px-3 d-flex w-100 h-50 mt-4 pb-5"
+               class="col-sm-2 d-flex w-100 h-50"
                aria-label="Details card">
         <span  class="ms-4 me-4 text-center">
           <h1>{{ totalPoolUsedBytes | dimlessBinary}}</h1>
         </span>
       </cd-card>
       <cd-card cardTitle="Avg Object Size"
                i18n-title
-               class="col-sm-2 px-3 d-flex w-100 h-50 mt-2 pb-5"
+               class="col-sm-2 d-flex w-100 h-50 pt-2"
```

Comment:
Why do we have a extra class `pt-2` which in this and not in used capacity? 
```suggestion
               class="col-sm-2 d-flex w-100 h-50 pt-2"
```

Reply:
Yes, you are right..updated the PR, added equal padding to used capacity as well